applications/monitoring/values.yaml

```diff
@@ -77,20 +189,83 @@ cronjob:
     # -- taskmaker schedule
     taskmaker: "6-59/15 * * * *"
 
+# -- Gafaelfawr ingress for InfluxDBv2, only used if the service is enabled.
```

Comment:
Some of the individual settings are documented and you also have documentation at the top level. Better to pick one or the other, since I don't know what helm-docs will make of that. Documenting the individual parameters is probably better since all but one is documented already.